ldap/client/api/src/main/java/org/apache/directory/ldap/client/api/LdapNetworkConnection.java

```diff
@@ -516,31 +522,59 @@ private static LdapConnectionConfig buildConfig( String server, int port, boolea
     }
 
 
+    //-------------------------- The methods ---------------------------//
     /**
      * Create the connector
      */
-    private void createConnector() throws LdapException
+    private IoConnector createConnector() throws LdapException
     {
         // Use only one thread inside the connector
-        connector = new NioSocketConnector( 1 );
+        IoConnector conn = new NioSocketConnector( 1 );
```

Comment:
there is already a class field called `connector`; is there a reason to create a local variable for it?